world.js

```diff
@@ -301,8 +301,11 @@ const _getBindSceneForRenderPriority = renderPriority => {
     case 'low': {
       return sceneLowPriority;
     }
-    case 'postScene': {
-      return postScene;
+    case 'perspectiveScene': {
```

Comment:
This can be renamed to `postPerspectiveScene` and similar, to indicate its actual order.